frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import {
+  StateCounts,
+  Severity,
+  AlertState,
+  alertsRowFilters,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import { alertDescription, alertState, alertSeverity } from '@console/internal/reducers/monitoring';
+import { Alert, Rule } from '@console/internal/components/monitoring/types';
+import { YellowExclamationTriangleIcon } from '@console/shared';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+export const monitoringAlertColumn = [
+  {
+    title: 'Name',
+    cellFormatters: [expandable],
+    transforms: [sortable],
+  },
+  { title: 'Severity', transforms: [sortable, cellWidth(20)] },
+  { title: 'Alert State', transforms: [sortable, cellWidth(20)] },
+  '',
+];
+
+export const monitoringAlertRows = (alertrules: Rule[]) => {
+  const rows = [];
+  _.map(alertrules, (rls) => {
+    rls.type !== 'recording' &&
```

Comment:
Nit: Probably better to use `rls.type !== 'alerting'` than `rls.type !== 'recording'` to be clear that we only handle `alerting` types. (Although we expect to have `alerting` and `recording` currently.)

Reply:
You mean `rls.type === 'alerting'` instead of `rls.type !== 'recording'`?

Reply:
Yes, that's what I meant